.travis.yml

```diff
@@ -1,5 +1,5 @@
 dist: trusty
-sudo: false
+sudo: true
```

Comment:
Requiring `sudo` just to install opencv into `/usr/local` while you can install it anywhere, for example `$HOME/local`, is distasteful. Especially when `pkg-config` is here to find the various bits you need afterwards. You will end up using heavier and slower executors, and using privileged commands without any good reason.